src/Components/Modal/Modal.tsx

```diff
@@ -0,0 +1,276 @@
+import React, { useState } from "react";
+import styles from "./Modal.module.css";
+import { useFetch } from "../../hooks/useFetch";
+import { BASE_URL, DEPLOY_URL } from "../../constants/baseURL";
+import Image from "next/image";
+
+interface ModalProps {
+    title: string;
+    subtitle?: string;
+    input?: boolean;
+    placeholder?: string;
+    list?: boolean;
+    btnText?: string;
+    btnColor?: "submit" | "delete";
+    share?: boolean;
+    folderId?: string;
+    onClose: () => void;
+    onSubmit?: (value: string) => void;
+}
+
+interface Link {
+    id: string;
+    created_at: string;
+    name: string;
+    user_id: number;
+    favorite: boolean;
+    link: {
+        count: number;
+    };
+}
+
+const Modal: React.FC<ModalProps> = ({
+    title,
+    subtitle,
+    input,
+    placeholder,
+    list,
+    btnText,
+    btnColor,
+    share,
+    folderId,
+    onClose,
+    onSubmit,
+}: ModalProps) => {
+    const [inputValue, setInputValue] = useState<string>("");
+    const folderList = useFetch(`${BASE_URL}users/1/folders`);
+    const [isError, setIsError] = useState<boolean>(false);
+    const url = `${DEPLOY_URL}shared?user=1&folder=${folderId}`;
+
+    const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+        setInputValue(e.target.value);
+    };
+
+    const handleBlur = () => {
+        if (inputValue.trim() !== "") {
+            setIsError(false);
+        } else setIsError(true);
```

Comment:
else 후 중괄호 정확하게 작성해주세요! 
```suggestion
        if (inputValue.trim() !== "") {
            setIsError(false);
        } else {
            setIsError(true);
        }
```